workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/MigrationPreprocessorActionNode.java

```diff
@@ -0,0 +1,71 @@
+package com.wipro.ats.bdre.wgen;
+
+import com.wipro.ats.bdre.md.beans.ProcessInfo;
+
+/**
+ * Created by cloudera on 3/31/16.
+ */
+
+/*
+Action nodes are the mechanism by which a workflow triggers the execution of a task
+Here, we set the id and return name of the action node.
+The method getXML() returns a string which contains name, Id, next success node(ToNode) and next failure node(TermNode)
+for the current action node, appropriately formatted as XML.
+*/
+
+    public class MigrationPreprocessorActionNode extends GenericActionNode {
+
+
+        private ProcessInfo processInfo = new ProcessInfo();
+        private ActionNode actionNode = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "actionNode" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)